pdk.yaml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
was the file added on purpose or should it actually be in .gitignore?

Reply:
Hi, it was added on purpose from pdk

Reply:
It's coming automatically with pdk update